language-plutus-core/exe/Main.hs

```diff
@@ -59,7 +61,9 @@ data EvalOptions = EvalOptions Input EvalMode
 type ExampleName = T.Text
 data ExampleMode = ExampleSingle ExampleName | ExampleAvailable
 newtype ExampleOptions = ExampleOptions ExampleMode
-data Command = Typecheck TypecheckOptions | Eval EvalOptions | Example ExampleOptions
+data PrintMode = Classic | Debug deriving (Show, Read)
+data PrintOptions = PrintOptions Input PrintMode
```

Comment:
Several of the other commands prettyprint programs, can we give this option to all of them?

Reply:
Then IMO we can just change the command name to be `parse`.

Reply:
I might want to add a few more actions later, so I'll think about reworking this then.  By "the command name" do you mean `print`?  I thought at first you meant `plc`.

Reply:
I mean, e.g. evaluation also produces a term, and so logically could take printing options.

Yeah, I meant `print` -> `parse` (since it basically just checks if it parses and then prints its). But `print` is also fine.